examples/qtmegachatapi/MegaChatApplication.cpp

```diff
@@ -185,7 +185,10 @@ void MegaChatApplication::onUsersUpdate(mega::MegaApi * api, mega::MegaUserList
             itContacts = this->mMainWin->contactWidgets.find(userHandle);
             if (itContacts == this->mMainWin->contactWidgets.end())
             {
-                mMainWin->addContact(userHandle);
+                if (user)
```

Comment:
You shouldn't need this check, you're iterating a list, items of that list should be always valid.